plugins/cache_range_requests/cache_range_requests.cc

```diff
@@ -111,24 +112,29 @@ create_pluginconfig(int argc, char *const argv[])
   --argv;
 
   for (;;) {
-    int const opt = getopt_long(argc, argv, "", longopts, nullptr);
+    int const opt = getopt_long(argc, argv, "i:", longopts, nullptr);
     if (-1 == opt) {
       break;
     }
 
     switch (opt) {
-    case 'p': {
-      DEBUG_LOG("Plugin modifies parent selection key");
-      pc->ps_mode = PS_CACHEKEY_URL;
-    } break;
     case 'c': {
-      DEBUG_LOG("Plugin considers the '%.*s' header", (int)X_IMS_HEADER.size(), X_IMS_HEADER.data());
+      DEBUG_LOG("Plugin considers the CRR ims header");
```

Comment:
Can this be reverted such that the default header is printed out when a custom header is not in use? Specifically, print the value of `DefaultImsHeader` here.

Reply:
The other 2 messages in the same function should cover it (based on override being used or not):

DEBUG_LOG("Plugin uses custom CRR ims header: %s", optarg);
DEBUG_LOG("Plugin uses default ims header: %s", pc->ims_header.c_str())